src/AppInstallerCommonCore/Public/winget/UserSettings.h

```diff
@@ -130,6 +133,8 @@ namespace AppInstaller::Settings
         SETTINGMAPPING_SPECIALIZATION(Setting::NetworkDOProgressTimeoutInSeconds, uint32_t, std::chrono::seconds, 60s, ".network.doProgressTimeoutInSeconds"sv);
         SETTINGMAPPING_SPECIALIZATION(Setting::InstallLocalePreference, std::vector<std::string>, std::vector<std::string>, {}, ".installBehavior.preferences.locale"sv);
         SETTINGMAPPING_SPECIALIZATION(Setting::InstallLocaleRequirement, std::vector<std::string>, std::vector<std::string>, {}, ".installBehavior.requirements.locale"sv);
+        SETTINGMAPPING_SPECIALIZATION(Setting::InstallRoot, std::string, std::filesystem::path, {}, ".installBehavior.installRoot"sv);

```

Comment:
This should remain a string through settings, as both support for environment variable expansion and properly creating the directories will need to be handled with care (AKA not just jammed into settings).